### DISCUSSION TITLE
Change text of cell names/title in innerText names/title

### DISCUSSION THREAD

**Discussion by boomshakar** [GrapesJS/grapesjs#4274] [Q&A] [answered]
![Screenshot (154)](https://user-images.githubusercontent.com/51333883/164320592-21240e4f-6df8-41a3-abb6-b7b3c189a9ed.png)

Looking at the screenshot uploaded, i circled **Table Cell, Classes, Selected:**
NOTE: I'm currently using the newsletter preset plugin.

I'd like to change the intext/title of 'Table cell', or 'Table' to something else like 'Custome Table Name'
and also I want to change the innertext/title of 'Classes' & 'Selected' to something custom.
 Pleas how do I go about it??

T for Thanks🙏

**Comment by artf** [comment, marked as answer]
Check [i18n](https://grapesjs.com/docs/modules/I18n.html) page and use the [default EN file](https://github.com/artf/grapesjs/blob/master/src/i18n/locale/en.js) as a reference

**Reply by boomshakar** [reply]
Thank you very much @artf, yo  the best!,
I was able to achieve my result through :


      let I18nManager = edit.I18n;
      I18nManager.addMessages({
        en: {
          domComponents: {
            // overriding the default Table, Table Cell
            names: {
              table: 'AAAA',
              cell: 'BBBBB',
            },
          },
          selectorManager: {
            // overriding the default Classes, Selected
            label: 'CCCC',
            selected: 'DDDDD',
          },
        },
      });
